app/components/documents/document-preview/document-preview-modal.hbs

```diff
@@ -1,22 +1,31 @@
-<Auk::Modal @size="full-screen">
-  <Auk::Modal::Header @title={{this.selectedVersion.name}} @onClose={{@onClose}} />
-  <Auk::Modal::Body @noPadding={{true}}>
+<AuModal
+  @modalOpen={{true}}
+  @closeModal={{@onClose}}
+  @size="fullscreen"
+  @padding="none"
+>
+  <:title>
+    <span class="au-u-flex au-u-flex--vertical-center au-u-flex--spaced-small">
+      {{@piece.name}}
+      <AuLink href={{@piece.file.namedDownloadLink}} @skin="button-naked" @icon="download" @iconAlignment="left" download>Downloaden</AuLink>
```

Comment:
Download button verbergen indien file niet beschikbaar is voor de gebruiker (indien `@piece.file == null`)?

Reply:
Zouden we niet beter de hele document view route redirecten als de gebruiker het document niet mag/kan zien? :thinking:

Reply:
Misschien wel. Alhoewel hij (deel van) de metadata in sidepanel wel mag/moet kunnen zien, ook als hij het document zelf niet mag zien.